.github/workflows/validate_kong_image_trigger_via_label.yaml

```diff
@@ -14,6 +14,9 @@ jobs:
       ISSUE_NUMBER: ${{ github.event.issue.number }}
       GH_TOKEN: ${{ secrets.K8S_TEAM_BOT_GH_PAT }}
     steps:
+      - name: checkout repository
+        id: checkout_kic_repo
```

Comment:
I don't see `checkout_kic_repo` being used anywhere. Can we remove this from this file?

Reply:
Removed this part in all jobs.